safespace/src/App.js

```diff
@@ -11,10 +11,10 @@ import Footer from './components/Footer';
 function App() {
   return (
     <div className="App">
-      <Header />
-      <Route path='/login' component={Login} />
-      <Route exact path='/' component={SafeSpace} />
-      <Footer />
+        <Route exact path='/' component={Header} />
```

Comment:
you could have also specified the header and footer inside the PrivateRoute HOC instead of having to declare it in the '/' route in this file